### PR TITLE
add HomeWork7

### PR DESCRIPTION
**Pull request by KozyrickiS**

### CODE REVIEW

File: ConsoleHW7/AssemblyOne/Developer.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AssemblyOne
+{
+    class Developer : Employee
+    {
+        private const string position = "Developer";
```
Comment (VadzimPapko) [inline ConsoleHW7/AssemblyOne/Developer.cs:11]: All const variable should be started with Upper case letter, 
for example
private const string position => `private const string Position`

File: ConsoleHW7/AssemblyOne/Developer.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        public override void GetInfoAboutEmployment()
+        {
+            if ((lastname != null) && (name != null))
+            {
+                AddDeveloper();
+            }
+            else
+            Console.WriteLine($"Developer {lastname} {name} in company {CompanyNamePublic} since {dateOfEmployment}");
```
Comment (VadzimPapko) [inline ConsoleHW7/AssemblyOne/Developer.cs:19]: just add one TAB formatting there

File: ConsoleHW7/AssemblyOne/Employee.cs [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+{
+    public class Employee
+    {
+        public string name;
+        public string lastname;
+        public DateTime BirthDate;
+        public DateTime dateOfEmployment;
+        private const string position = "Employee";
+
+        public const string CompanyNamePublic = "ООО Печенька";
```
Comment (VadzimPapko) [inline ConsoleHW7/AssemblyOne/Employee.cs:17]: ahahha, great company names :)

File: ConsoleHW7/AssemblyTwo/Manager.cs [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AssemblyTwo
+{
+    class Manager : Employee
+    {
+        private const string position = "Manager";
```
Comment (VadzimPapko) [inline ConsoleHW7/AssemblyTwo/Manager.cs:12]: the same with naming

**Review by VadzimPapko — APPROVED** [review]
done

File: ConsoleHW7/AssemblyOne/Developer.cs (hunk 1 above)
Comment (KozyrickiS) [reply ConsoleHW7/AssemblyOne/Developer.cs:11]: all consts a renamed with Upper case latter

File: ConsoleHW7/AssemblyOne/Developer.cs (hunk 2 above)
Comment (KozyrickiS) [reply ConsoleHW7/AssemblyOne/Developer.cs:19]: fixed

**Pull request merged**